lib/sparse/src/index/inverted_index/inverted_index_compressed_mmap.rs

```diff
@@ -39,8 +39,12 @@ impl StorageVersion for Version {
 
 #[derive(Debug, Default, Clone, Serialize, Deserialize)]
 pub struct InvertedIndexFileHeader {
-    pub posting_count: usize, // number of posting lists
-    pub vector_count: usize,  // number of unique vectors indexed
+    /// Number of posting lists
+    pub posting_count: usize,
+    /// Number of unique vectors indexed
+    pub vector_count: usize,
+    /// Total size of all searchable sparse vectors in bytes
+    pub total_sparse_size: usize,
```

Comment:
We can't just add a required field to serializable structure. This will break backward compatibility

Reply:
I am curious, why are the data compatibility tests on CI not failing already for this PR?

Reply:
I just checked, the inverted index that is currently on the compatibility snapshots is pre 0.2.0 (compressed index), so it detects the version mismatch and just rebuilds from scratch. This is why it doesn't fail. 

However if we try to load a 0.2.0 index, this new field should make it fail.